gtests/net/mptcp/sockopts/sockopt_cork_nodelay.pkt

```diff
@@ -0,0 +1,50 @@
+--tolerance_usecs=100000
+`../common/defaults.sh`
+
+
++0.0  socket(..., SOCK_STREAM, IPPROTO_MPTCP) = 3
++0.0  setsockopt(3, SOL_SOCKET, SO_REUSEADDR, [1], 4) = 0
++0.0  setsockopt(3, SOL_TCP, TCP_CORK, [1], 4) = 0
+
++0    bind(3, ..., ...) = 0
++0    listen(3, 1) = 0
+
++0.0    <  addr[caddr0] > addr[saddr0]  S   0:0(0)               win 65535  <mss 1460, sackOK, TS val 4074410674 ecr 0,          nop, wscale 8, mpcapable v1 flags[flag_h] nokey>
++0.0    >                               S.  0:0(0)       ack 1              <mss 1460, sackOK, TS val 4074410674 ecr 4074410674, nop, wscale 8, mpcapable v1 flags[flag_h] key[skey]>
++0.0    <                                .  1:1(0)       ack 1   win 256    <nop, nop,         TS val 4074410674 ecr 4074410674,                mpcapable v1 flags[flag_h] key[ckey=2, skey]>
++0    accept(3, ..., ...) = 4
+
+// TCP_CORK synced from listener
++0.0  write(4, ..., 40) = 40
++0.05 write(4, ..., 40) = 40
+
+// clearing TCP_CORK should push pending bytes out
++0.01 setsockopt(4, SOL_TCP, TCP_CORK, [0], 4) = 0
```

Comment:
detail about the call side: may you add one more space after each time to have at least two spaces between the time and the next column?
While at it, the two `<nop,` below are not aligned with all the others :)

Reply:
Thank you for the review!
Could you tell me if the commit adressing all the changes should be force-pushed, or just it will be squashed on merge?

Reply:
Please force-push your new version to keep only "good commits". Even if it is not as good as with Gerrit, we can easily see the differences between two force-pushed versions on Github so that side is not an issue :)